<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistFilterWidget</class>
 <widget class="QWidget" name="PlaylistFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>117</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset theme="text-richtext"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="2">
    <widget class="QPushButton" name="buttonClear">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Clear</string>
     </property>
     <property name="icon">
      <iconset theme="edit-clear"/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="editSearch"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QComboBox" name="comboType">
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Radio</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QComboBox" name="comboLanguage">
     <property name="maxVisibleItems">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QComboBox" name="comboCategory">
     <property name="maxVisibleItems">
      <number>50</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonClear</sender>
   <signal>clicked()</signal>
   <receiver>editSearch</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
